### PR TITLE
Get the number from the first line

### PR DESCRIPTION
**Pull request by abunsen**

### CODE REVIEW

**Comment by RodneyPerez** [comment]
we should save it in a variable. and trim the white space and make it into an integer

File: answer.rb [hunk 1]
```diff
@@ -0,0 +1 @@
+gets
```
Comment (rbarrar) [inline answer.rb]: I would suggest saving it in a variable

Comment (RodneyPerez) [inline answer.rb]:
```
n = gets.chomp.to_i
```

Comment (readytocode123) [inline answer.rb]: Fine....

**Pull request merged**